### DISCUSSION TITLE
Issue deploying MERN app

### DISCUSSION THREAD

**Discussion by AlanM03** [vercel/community#3446] [Help]
### Summary

Hello, I finished creating a MERN app that I've been working on over the summer, and whenever I try to deploy to vercel I get this weird error. "s.at is not a function. (In 's.at(-1)', 's.at' is undefined)". I know you probably hear this all the time but the project is fully functional on my device database and all. I included all env files as well as the following code in a vercel json file but still issues.

{
    "version":2,
    "builds": [
        { "src": "*.js", "use": "@vercel/node" }
    ],
    "routes": [
        {
            "src": "/(.*)",
            "dest": "/"
        }
    ]
}

<img width="1365" alt="Screen Shot 2023-07-31 at 1 03 34 PM" src="https://github.com/vercel/community/assets/85244851/b99b963e-6db5-4ced-9838-a895ec1047a7">

### Steps to Reproduce

To reproduce this I just deploy my backend GitHub repo on the vercel site and it happens every time

**Comment by amyegan** [comment]
Hey @AlanM03. If you can share your public repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), then someone here may be able to recognize what is going wrong. 🙏
